data/json/items/ammo.json

```diff
@@ -946,6 +946,7 @@
         "category" : "chems",
         "price" : 3500,
         "name" : "sprayable fungicide",
+        "name_plural" : "sprayable fungicide",
```

Comment:
A proper fix is to exclude plural names of items stored by charges from the scope of work for translators. No need to burden them by useless work. I'm not saying I'll not merge this PR, but it looks like excessive work.